docs/getting-started.md

```diff
@@ -0,0 +1,168 @@
+![code-snippet-select-all](../_images/code-snippet-select-all.png)
+
+### Search
+As described in the [Gro Data: Key Concepts](../docs/gro-data-key-concepts) page, a data series in Gro is a unique combination of the entities: item, metric, region, partner_region (optional), frequency, and source. To find the specific entity you would like to retrieve data for, you can use a variety of search methods. For example, `client.search` will return a list of IDs that match your search term. If you want to understand the differences between various search results, you may find the `client.search_and_lookup` method more helpful.
+
+`client.search_and_lookup('items','Corn')` will return a list of all items that contain "corn" in their name, along with supporting information like id, name, and ids of other items contained by a given item.
+
+Note: the above query will return a generator object. Running a query such as: 
+```for result in client.search_and_lookup('items','corn'): print(result)```
+should help produce a list of all the relevant information.
+
+### Get data series
+Instead of searching for all the individual entity IDs required to create a data series, the `client.get_data_series` method will return a list of all the data series available for the filters you have supplied. For example, if you are interested in Russian Oats you could use the following code to find out all the available data series that have "Oats" (item_id = 327) as the item and "Russia" (region_id = 1168) as the region:
+
+```client.get_data_series(**{'item_id': 327, 'region_id': 1168})```
+
+### Lookup contains
+Our ontology is defined in terms of a graph, with metrics/items/regions containing others. In each case, you can see the `'contains'` property in the output of `client.lookup(type, id)`. For example:
+
+```client.lookup('items', 10009)['contains'] ```
+
+will return a list of items ids for items that are cereals (item_id = 10009): `[..., 274, 422, ...]`. Once you have those ids, you can use the `client.lookup()` function on each one to find more info, like their names, e.g.: `client.lookup('items', 274)['name']` will return `Corn`.
+
+Similarly, for regions, `client.lookup('regions', 1215)[contains]` will return a list of region ids for regions that are in the US (region_id = 1215): `[13100, 13061, 13053, 13099, ....]`. And each of those can be further looked up e.g. `client.lookup('regions', 13100)` will return `{'name': 'Wisconsin', 'level': 4, 'contains': [139839, 139857, 139863, ...]}`.
+
+### Get descendants
+The Lookup Contains method can return a list of an entity's direct children. However, you may wish to discover all of the lower level regions that belong to a high level region. For this special case of traversing regions, there's a shortcut function that does it more directly and gives the option of filtering by region level: `get_descendant_regions(region_id, descendant_level)`.
+
+Where:
+```
+provinces_of_brazil = client.get_descendant_regions(1029, 4)

```

Comment:
Instead of using the numbers directly, I like to import the names and use those instead:

```py
from api.client.lib import REGION_LEVELS
provinces_of_brazil = client.get_descendant_regions(1029, REGION_LEVELS['province'])
```

But I think also we can just improve `get_descendant_regions()` to take either an integer or a string for "level" so you could do

```py
provinces_of_brazil = client.get_descendant_regions(1029, 'province')
```

Simple change to make

Reply:
Done.